spec/03-plugins/37-opentelemetry/04-exporter_spec.lua

```diff
@@ -52,17 +52,22 @@ for _, strategy in helpers.each_strategy() do
         port = helpers.mock_upstream_port,
       })
 
-      bp.routes:insert({ service = http_srv,
-                         protocols = { "http" },
-                         paths = { "/" }})
+      local route = assert(bp.routes:insert({ service = http_srv,
+                                              protocols = { "http" },
+                                              paths = { "/" }}))
 
-      bp.plugins:insert({
+      assert(bp.routes:insert({ service = http_srv,
+                                protocols = { "http" },
+                                paths = { "/no_plugin" }}))
+
+      assert(bp.plugins:insert({
         name = "opentelemetry",
+        route = route,
```

Comment:
Do not apply otel plugin on the route by default, add an arg in `setup_instrumentations` to configure it instead.